docs/source/about/showcase.rst

```diff
@@ -0,0 +1,18 @@
+.. _showcase:
+
+Community Showcase
+==================
+
+.. list-table::
+   :widths: 25 50 25
+   :header-rows: 1
+
+   * - Title
+     - Short Description
+     - User
+   * - `sktime: a unified python library for time series machine learning <https://towardsdatascience.com/sktime-a-unified-python-library-for-time-series-machine-learning-3c103c139a55>`_
+     -
+     - :user:`Alexandra Amidon <lynnssi>`_
```

Comment:
Hey @afzal442 - the users in the table don't render correctly: https://sktime--1337.org.readthedocs.build/en/1337/about/showcase.html
```suggestion
     - :user:`lynnssi`
```

Reply:
this should also work
```
:user:`Alexandra Amidon <lynnssi>`
```